### PR TITLE
[RAPPS] French translation update following PR 2792

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- French translation update following PR #2792 (specified database source)

### CODE REVIEW

File: base/applications/rapps/lang/fr-FR.rc [hunk 1]
```diff
@@ -245,5 +245,5 @@ STRINGTABLE
 BEGIN
     IDS_DL_DIALOG_DB_DISP "Base de données d'applications"
     IDS_DL_DIALOG_DB_DOWNLOAD_DISP "Mise à jour de la base de données…"
-    IDS_DL_DIALOG_DB_UNOFFICIAL_DOWNLOAD_DISP "Updating Database… (Unofficial source)"
+    IDS_DL_DIALOG_DB_UNOFFICIAL_DOWNLOAD_DISP "Mise à jour de la base de données… (No officielle)"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:248]:
```suggestion
    IDS_DL_DIALOG_DB_UNOFFICIAL_DOWNLOAD_DISP "Mise à jour de la base de données… (Non officielle)"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 2]
```diff
@@ -228,7 +228,7 @@ STRINGTABLE
 BEGIN
     IDS_LICENSE_OPENSOURCE "Open Source"
     IDS_LICENSE_FREEWARE "Freeware"
-    IDS_LICENSE_TRIAL "Trial/Demo"
+    IDS_LICENSE_TRIAL "Version d'essai/Demo"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:231]:
```suggestion
    IDS_LICENSE_TRIAL "Version d'essai / Démo"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 3]
```diff
@@ -193,14 +193,14 @@ BEGIN
...
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "L'URL spécifiée est invalide ou non-supportée. Veuillez corriger."
     IDS_APP_REG_REMOVE "Êtes-vous sûr de vouloir supprimer du registre les données du programme installé ?"
     IDS_INFORMATION "Information"
     IDS_UNABLE_TO_DOWNLOAD "Impossible de télécharger le paquet : adresse introuvable !"
     IDS_UNABLE_TO_DOWNLOAD2 "Impossible de télécharger le paquet : vérifiez votre connexion à Internet !"
     IDS_UNABLE_TO_REMOVE "Impossible de supprimer du registre les données du programme !"
     IDS_UNABLE_TO_INSTALL "Impossible d'ouvrir l'installeur !"
-    IDS_UNABLE_TO_QUERY_CERT "Unable to retrieve certificate info.\n\nDo you want to continue anyway?"
+    IDS_UNABLE_TO_QUERY_CERT "Impossible de récupérer les informations de certificat.\n\nVoulez vous continuer ?"
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:203]:
```suggestion
    IDS_UNABLE_TO_QUERY_CERT "Impossible de récupérer les informations de certificat.\n\nVoulez-vous continuer ?"
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 4]
```diff
@@ -193,14 +193,14 @@ BEGIN
     IDS_APPLICATIONS "Applications"
     IDS_CHOOSE_FOLDER_TEXT "Choisissez un dossier dans lequel les programmes seront téléchargés :"
     IDS_CHOOSE_FOLDER_ERROR "Le dossier que vous avez spécifié n'existe pas. Le créer ?"
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "L'URL spécifiée est invalide ou non-supportée. Veuillez corriger."
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:196]:
```suggestion
    IDS_URL_INVALID "L'URL spécifiée est invalide ou non prise en charge. Veuillez la corriger."
```

File: base/applications/rapps/lang/fr-FR.rc [hunk 5]
```diff
@@ -66,9 +66,9 @@ BEGIN
     EDITTEXT IDC_DOWNLOAD_DIR_EDIT, 15, 86, 166, 12, WS_CHILD | WS_VISIBLE | WS_GROUP | ES_AUTOHSCROLL
     PUSHBUTTON "&Sélectionner", IDC_CHOOSE, 187, 85, 50, 14
     AUTOCHECKBOX "&Supprimer l'installateur du programme après l'installation", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
-    GROUPBOX "Software source", -1, 4, 118, 240, 46
-    CONTROL "Use default", IDC_SOURCE_DEFAULT, "Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 15, 132, 74, 10
-    CONTROL "Specified source", IDC_USE_SOURCE, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 15, 147, 74, 10
+    GROUPBOX "Source des logiciels", -1, 4, 118, 240, 46
```
Comment (HBelusca) [inline base/applications/rapps/lang/fr-FR.rc:69]:
```suggestion
    GROUPBOX "Source des programmes", -1, 4, 118, 240, 46
```
(Everywhere else in RAPPS we talk about "programmes", not "logiciels".)

**Comment by KyleKatarn31** [comment]
Thank you @HBelusca . All suggested changes are now committed.

**Pull request merged**